pkg/object/httpserver/mux.go

```diff
@@ -175,16 +176,19 @@ func (mr *muxRule) pass(ctx context.HTTPContext) bool {
 }
 
 func (mr *muxRule) match(ctx context.HTTPContext) bool {
-	r := ctx.Request()
+	host := ctx.Request().Host()
+	if h, _, err := net.SplitHostPort(host); err == nil {
+		host = h
+	}
```

Comment:
please move this code snippet after the first `if` to improve performance.

Reply:
Agreed, if `mr.host` is nil, there is not need to pay the cost for calling `net.SplitHostPort` function.